src/ingest_validation_tools/upload.py

```diff
@@ -146,28 +144,28 @@ def get_errors(self, **kwargs) -> ErrorDict:
         self.validation_routine()
         self._get_reference_errors()
 
-        # Plugin error checking is costly, by default this bails
+        # Plugin error checking is costly, this bails
         # if other errors have been found already
-        if self.errors and not self.run_plugins:
+        if self.errors:
             self.errors.plugin_skip = (
                 "Skipping plugins validation: errors in upload metadata or dir structure."
             )
-        elif self.run_plugins:
+        else:
```

Comment:
@gesinaphillips - I think we only want to run plugin validation if the self.run_plugins is true. I think the issue with the previous logic was that if the self.run_plugins is true it wouldn't fall into the self.errors if statement. So I think the resolution here is to bring the `elif self.run_plugins` back and leave the `and not self.run_plugins` off the `self.errors` check.

Reply:
Oh I think I see the reason for my confusion. My initial implementation of `run_plugins` was meant to let you run plugin validation _even if_ there were upstream errors (I think the idea was that there might be cases where you want to get ALL errors). However given the size of most uploads that is probably not needed. 

I believe this change (fail if errors, skip plugins unless explicitly passed `run_plugins=True`) would require us to either: set `run_plugins` to default True or add a run_plugins arg to the validate_upload.py DAG (and maybe others?).

Reply:
I guess this purpose makes the original `elif self.run_plugins` redundant at best, admittedly.

Reply:
ANYWAY decision point:
- Leave as-is (change `elif self.run_plugins` to just `else` I think): default logic is to set `run_plugins` to False so it fails on any errors, and only run plugins if no errors OR if run_plugins explicitly set to True. Can run plugins if there are errors but requires passing `run_plugins=True`.
- Change A: change `if self.errors and not self.run_plugins` to just `if self.errors`: default logic is that run always fails if there are errors, and plugins only run if `run_plugins` is explicitly set to True. Change needed in ingest-pipeline.
- Change B: change `if self.errors and not self.run_plugins` to just `if self.errors` and default parameter to `run_plugins=True`: default logic is that run always fails if there are errors, but plugins always run unless there are errors (and unless `run_plugins` is explicitly set to False). No change to ingest-pipeline.